Server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+// define the ping route
+app.get("/ping",(req,res)=>{
```

Comment:
Consider adding error handling for your server setup and routes. This can help prevent unhandled exceptions and improve the stability of your application. [important]